build/litsearch/getInternalData.py

```diff
@@ -267,6 +282,12 @@ def mergePaperDataFrames(dataFramesList):
         dataFramesList_DOI.append(df[validDOI])
         dataFramesList_NoDOI.append(df[invalidDOI])
 
+    # Check if there are issues with duplicate DOIs
+    for df in dataFramesList_DOI:
+        duplicateDOIs = getDuplicates(list(df.index))
```

Comment:
The pandas `duplicated` function could be used to get duplicate rows. However, this is single-purpose code, so I don't think it matters whether we use that or a specialized function.